omod/src/main/webapp/resources/htmlforms/073-ChildRegister.xml

```diff
@@ -0,0 +1,497 @@
+                    </p>
+                </td>
+                <td >
+                    <p>
+                        <label>M.U.A.C(cm)</label>
+                        <obs conceptId="1343" />
+                    </p>
+                </td>
+
+                <td>
+                    <p>
+                        <label>M.U.A.C</label>
+                        <includeIf velocityTest="$patient.age &lt; 5">
+                            <obs conceptId="99030"
+                                 answerConceptIds="99028,99029,99027"
+                                 answerLabels="Red &lt; 11.5cm, Yellow 11.5cm - &lt;12.5cm,Green &gt;= 12.5cm" id="muac-qualitative" />
+                        </includeIf>
+                        <includeIf velocityTest="$patient.age &lt; 10 &amp;&amp; $patient.age &gt;=5">
+                            <obs conceptId="99030"
+                                 answerConceptIds="99028,99029,99027"
+                                 answerLabels="Red &lt; 13.5cm,Yellow 13.5cm - &lt;14.5cm,Green &gt;=14.5cm" id="muac-qualitative" />
+                        </includeIf>
+                        <includeIf velocityTest="$patient.age &lt; 18 &amp;&amp; $patient.age &gt;=10">
+                            <obs conceptId="99030"
+                                 answerConceptIds="99028,99029,99027"
+                                 answerLabels="Red &lt; 16.5cm, Yellow 16.5cm - &lt;19.0cm,Green &gt;=19.0cm" id="muac-qualitative" />
+                        </includeIf>
+                        <includeIf velocityTest="$patient.age &gt;=18">
+                            <obs conceptId="99030"
+                                 answerConceptIds="99028,99029,99027"
+                                 answerLabels="Red &lt;19.0cm , Yellow 19.0cm - &lt;22.0cm,Green &gt;=22.0cm" id="muac-qualitative" />
+                        </includeIf>
+
+                    </p>
+                </td>
+
+                <td colspan="2">
+                    <p>
+
+                        <label>INR No</label>
+                    </p>
+                    <obs conceptId="165027" />
+                </td>
+
+            </tr>
+
+
+        </table>
+    </fieldset>
+
+        <fieldset>
+
+            <legend>Immunization and IYCF Summary</legend>
+            <table width="100%" border="0" cellspacing="0" cellpadding="0">
+                <tr>
+                    <td>
+                        <p>
+                            <label>Fully Immunized by 1 Year</label>
+                            <obs conceptId="165031" answerConceptIds="1065, 1066" answerLabels="Yes, No" style="radio"
+                                 class="horizontal-layout-options"/>
+                        </p>
+                    </td>
+                    <td>
+                        <p>
+                            <label>IYCF Counselling</label>
+                            <obs conceptId="99051" answerConceptIds="1065, 1066"
```

Comment:
@faithNassiwa for boolean concepts use a checkbox not radio button as this does not work 
<img width="992" alt="screen shot 2018-04-26 at 12 55 53" src="https://user-images.githubusercontent.com/689900/39299422-56be6d30-4951-11e8-9883-262ab7dbc957.png">

Reply:
There is no style radio for boolean concepts see https://wiki.openmrs.org/display/docs/HTML+Form+Entry+Module+HTML+Reference#HTMLFormEntryModuleHTMLReference-Boolean

Reply:
Noted.